src/nifgen/examples/nifgen_trigger_example.py

```diff
@@ -0,0 +1,20 @@
+import nifgen
+import time
+
+with nifgen.Session('PXI1Slot12', [0,1]) as dev1, nifgen.Session('PXI1Slot16', [0,1]) as dev2:
```

Comment:
These examples only run on the 2 channel generators. What do you gain by specifying all channels? Isn't it the same to not specify (defaults to all)?